tests/lib/rules/html-indent.js

```diff
@@ -281,6 +281,72 @@ tester.run('html-indent', rule, loadPatterns(
           </pre>
         </template>
       `
+    },
+    {
+      filename: 'test.vue',
+      code: unIndent`
+        <template>
+          <pre>
+        <span>aaa</span>
+          <span>bbb</span>
+            <span>ccc</span>
+          </pre>
+        </template>
+      `
+    },
+    {
+      filename: 'test.vue',
+      code: unIndent`
+        <template>
+          <pre>aaa
+        bbb ccc
+        ddd
+        fff</pre>
```

Comment:
Shouldn't we also indent end tag properly?

Reply:
I think it is not good to indent end tags. Because indenting the end tag affects the contents.